### PR TITLE
chore(functions): increase retry counts for flaky tests

### LINKED ISSUE #5250: functions.ExampleIT: helloHttp_shouldRunWithFunctionsFramework failed
This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: 6ba0025078a96bcc0646dd74fe4fdf36451cb4c1
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/ed65fc38-8416-4146-ba33-be937319a9ed), [Sponge](http://sponge2/ed65fc38-8416-4146-ba33-be937319a9ed)
status: failed
<details><summary>Test output</summary><br><pre>com.google.common.truth.AssertionErrorWithFacts: 
expected: Hello world!
but was an empty string
	at functions.hello.http@1.0.22/functions.ExampleIT.helloHttp_shouldRunWithFunctionsFramework(ExampleIT.java:89)
</pre></details>

### PR DESCRIPTION
**Pull request by ace-n**
Fixes #5250 and #5305

### CODE REVIEW

**Review by lesv — APPROVED** [review]

File: functions/helloworld/hello-pubsub/pom.xml [hunk 1]
```diff
@@ -145,6 +145,7 @@
         <version>0.9.7</version>
         <configuration>
           <functionTarget>functions.HelloPubSub</functionTarget>
+          <port>8083</port>
```
Comment (averikitsch) [inline functions/helloworld/hello-pubsub/pom.xml:148]: Should we be hard coding this value, even if it's for the tests? Does this change how a user will deploy this sample?

Comment (ace-n) [reply functions/helloworld/hello-pubsub/pom.xml:148]: It shouldn't change deployment or test-running.

(cl/377974756 is the only thing changed by this PR.)

File: functions/helloworld/hello-pubsub/src/test/java/functions/ExampleIT.java [hunk 2]
```diff
@@ -60,13 +65,21 @@ public static void setUp() throws IOException {
 
     // Emulate the function locally by running the Functions Framework Maven plugin
     emulatorProcess = new ProcessBuilder()
-        .command("mvn", "function:run")
+        .command("/usr/local/bin/mvn", "function:run")
```
Comment (averikitsch) [inline functions/helloworld/hello-pubsub/src/test/java/functions/ExampleIT.java:68]:
```suggestion
        .command("mvn", "function:run")
```

File: functions/helloworld/hello-pubsub/pom.xml (hunk 1 above)
Comment (averikitsch) [reply functions/helloworld/hello-pubsub/pom.xml:148]: I see, can you add a comment for the user on why it's there

File: functions/helloworld/hello-pubsub/src/test/java/functions/ExampleIT.java (hunk 2 above)
Comment (ace-n) [reply functions/helloworld/hello-pubsub/src/test/java/functions/ExampleIT.java]: Good catch - meant to revert this before committing. 😢

File: functions/helloworld/hello-pubsub/pom.xml (hunk 1 above)
Comment (ace-n) [reply functions/helloworld/hello-pubsub/pom.xml:148]: I added one to the sample itself (already).

Bearing that in mind, did you *also* want me to add something to `pom.xml`?

Comment (averikitsch) [reply functions/helloworld/hello-pubsub/pom.xml:148]: Ah I see. lgtm

**Review by averikitsch — APPROVED** [review]

**Pull request merged**